e2e/suite1/specs/select-focus.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { e2e } from '@grafana/e2e';
+
+e2e.scenario({
+  describeName: 'Select focus/unfocus tests',
+  itName: 'Tests select focus/unfocus scenarios',
+  addScenarioDataSource: false,
+  addScenarioDashBoard: false,
+  skipScenario: false,
+  scenario: () => {
+    e2e.flows.openDashboard('5SdHCadmz');
+    e2e.pages.Dashboard.Toolbar.toolbarItems('Dashboard settings').click();
+
+    e2e.components.FolderPicker.container()
+      .should('be.visible')
+      .within(() => {
+        e2e.components.Select.input()
+          .wait(2000)
```

Comment:
Don't think we're doing the test right if we need to wait because it might become flacky? But I see how our e2e framework lacks a way to extend the retry period.

Reply:
It is waiting for the folder list to load. Can we do it in any other way?